webapp-django/crashstats/settings/base.py

```diff
@@ -586,25 +587,14 @@ def filter(self, record):
 # Comma-separated list of urls that serve version information in JSON format
 OVERVIEW_VERSION_URLS = config('OVERVIEW_VERSION_URLS', '')
 
-# Raven sends errors to Sentry.
-# The release is optional.
+# Sentry aggregates reports of uncaught errors and other events
 SENTRY_DSN = config('SENTRY_DSN', '')
 if SENTRY_DSN:
-    RAVEN_CONFIG = {
-        'dsn': SENTRY_DSN,
-        'release': SOCORRO_REVISION,
-        # Defines keys to be sanitized by SanitizeKeysProcessor
-        'sanitize_keys': [
-            'sessionid',
-            'csrftoken',
-            'anoncsrf',
-            'sc',
-        ],
-        'processors': (
-            'raven.processors.SanitizeKeysProcessor',
-            'raven.processors.SanitizePasswordsProcessor',
-        )
-    }
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        release=SOCORRO_REVISION,
+        send_default_pii=True,
+        integrations=[DjangoIntegration()])
```

Comment:
We should try to maintain raven's client side filtering to avoid accidentally collecting PII.

We have audited Sentry for storing PII (part of the reason we self-host it), can filter it server side, and can probably purge PII if we do collect it, but not collecting it in the first place would be best.

Looking at https://github.com/getsentry/sentry-python/issues/211 it looks like there isn't an equivalent for the raven filters and processors and we'd need to handle that in a `before_send` hook (looks like the raven filters are matching on field names and value regexes https://github.com/getsentry/raven-python/blob/master/raven/processors.py#L165).

Reply:
jwhitlock pointed out some additional nuance, so I'm going to run this by the rest of the secops team.

Reply:
Perfect timing on bringing this up right before the secops team meeting! So our consensus was that if we can filter out more PII client side we should do that, but it's also OK to continue collecting the same user fields since crash reports already contain user submitted PII and we need to associate users with their crashes.

And if the new sdk does less filtering we should try to preserve the behavior of raven's filters.

Reply:
I just chatted with @g-k and he pointed out that web users don't opt-in to webapp crash reports to sentry and so they're not really the same thing as firefox crash reports to socorro.

I've been on Socorro for like 3 years and I don't think I've looked at any of those fields before. Sure, past is not prologue, but I think that's probably a good indicator.

I vote we go the easy route and do `set_default_pii=False`. If that turns out to be a problem in the future, we can revisit.

Reply:
OK some things from further discussion with willkg:

* "webapp crashes -> sentry are really different than firefox crashes -> socorro" (especially in that they aren't opt-in)

* socorro usually goes out of its way to **not** associate Fx crashes with users (as in it's a legal or data requirement), so that's not a reason to collect PII

* willkg can't recall when he last used the PII fields so we can set that flag to false to not collect them

* we should re-implement the sanitizers from raven to preserve existing behavior (and upstream the change if sentry is open to it)